lesson002/task5/css/maine.css

```diff
@@ -0,0 +1,73 @@
+.container {
+    min-width: 100%;
+    max-width: 100%;
+    min-height: 50%;
+    max-height: 50%;
+    padding-right: 0;
+    padding-left: 0;
+}
+.carousel-control-next, .carousel-control-prev {
+    max-width: 3%;
+}
+a.bg-secondary:focus, a.bg-secondary:hover, button.bg-secondary:focus, button.bg-secondary:hover {
+    background-color: #7978789b!important;
+}
+.carousel-caption {
+    max-width: 300px;
+    left: 2%;
+    bottom: 70px; 
+}
+.btn {
+    display:flex;
+    width: max-content;
+    margin-left: 55px;
+    
+    
+}
+h3 {
+    text-align: left;
+    margin-left: 55px;
+}
+input {
+  margin-left: 25%;
+}
+footer .col-12, footer .col-6, .h33 {
+    text-align: center;
+}
```

Comment:
.h33      Do you like this class name yourself?

Reply:
Done